pyupgrade.py

```diff
@@ -1127,14 +1144,36 @@ def _is_six(self, node, names):
         return (
             isinstance(node, ast.Name) and
             node.id in names and
-            node.id in self._six_from_imports
+            node.id in self._from_imported_names['six']
         ) or (
             isinstance(node, ast.Attribute) and
             isinstance(node.value, ast.Name) and
             node.value.id == 'six' and
             node.attr in names
         )
 
+    def _is_oserror_complex_candidate(self, node):
+        # type: (ast.Call) -> bool
+        if _starargs(node):
```

Comment:
it's actually probably fine to allow starargs here -- we aren't rewriting any of the arguments

Reply:
sure